Finmer.Editor/FormMain.cs

```diff
@@ -281,6 +281,19 @@ private bool SaveAs()
             // Present a save dialog
             if (dlgSave.ShowDialog() != DialogResult.OK)
                 return false;
+                
+            // Pull the directory we're saving in, see if there's any other finmer projects already there.
+            string save_path = Path.GetDirectoryName(dlgSave.FileName);
+            string[] file_entries = Directory.GetFiles(save_path, "*.fnproj");
+            if (file_entries.Length >= 1 && !file_entries[0].Equals(dlgSave.FileName))
+            {
+                // Alert user that saving here will delete unused json files in the directory, have them acknowledge that fact before continuing.
+                DialogResult dr = MessageBox.Show("The selected folder contains another Finmer project. This is not supported by the Editor: files from the other project will be overwritten and/or deleted. Continuing anyway may cause unrecoverable data loss. Are you sure you wish to proceed?", "Finmer Editor", MessageBoxButtons.YesNo, MessageBoxIcon.Warning);
+                if (dr == DialogResult.No)
+                    return false;
+
+                return true;
```

Comment:
As mentioned, just let control flow leave this block. Returning `true` here will stop the save operation and report that it succeeded, which is incorrect since nothing was saved. (We only finished choosing the path to save to.)